docs/variants/novacustom_nv4x_adl/openness-score.md

```diff
@@ -1,17 +1,17 @@
 # Dasharo Openness Score
 
-Openness Score for novacustom_nv4x_adl_v1.7.1_full.rom
+Openness Score for novacustom_nv4x_adl_v1.7.2_full.rom
```

Comment:
@mkopec try to adapt the same format as I used here: https://github.com/Dasharo/docs/pull/686/files
(the version is the second heading with level 2 (##)). Then the other headings are one level lower, so that the version number can be navigated via TOC on the right. Note that Openness Score reports for new releases will also be added here, so this file has to be easily navigatable.

Reply:
https://github.com/Dasharo/docs/pull/729/commits/5ca84e96b5089e43fa73fd1f1bcc1d1952263d10